aiohttp/helpers.py

```diff
@@ -974,6 +975,9 @@ def set_cookie(
         if samesite is not None:
             c["samesite"] = samesite
 
+        if partitioned is not None:
+            c["partitioned"] = partitioned
```

Comment:
Can we also get a quick test to cover this? Can add a `@pytest.mark.skipif(sys.version_info < (3, 14), reason="No partitioned support")` to run only on 3.14.